### ISSUE TITLE
Multipart communication protocol examples do not always contain a "@context" in their header

### ISSUE THREAD

**Issue by matgnt** [International-Data-Spaces-Association/IDS-G#46]
Is there any reason that the following header for the DescriptionResponseMessage does NOT contain the typical JSON-LD context?
I thought this was mandatory:

https://github.com/International-Data-Spaces-Association/IDS-G/blob/main/Communication/protocols/multipart/README.md

```
--msgpart
Content-Disposition: form-data; name="header"
Content-Type: application/json
Content-Length: 409

{
  "@type" : "ids:DescriptionResponseMessage",
  "issued" : {
      "@value" : "2019-12-21T13:11:14.596+02:00",
      "@type" : "http://www.w3.org/2001/XMLSchema#dateTimeStamp"
    },
  "issuerConnector" : {"@id" : "https://broker.ids.isst.fraunhofer.de/" },
  "correlationMessage" : {"@id" : "https://w3id.org/idsa/autogen/selfDescriptionRequest/b0731661-7df1-43e5-bb75-50f0709f31c9" },
  "modelVersion" : "4.1.0",
  "@id" : "https://w3id.org/idsa/autogen/selfDescriptionResponse/851e3218-2bb7-45f9-8795-7f99c1f19680"
}
```

There is at least a second example with the same missing context.

**Comment by sebbader** [comment]
Hello @matgnt, 
thanks for pointing this out! Even though JSON-LD can omit the context element if only full IRIs are used, some of the examples indeed use prefixes or base attributes and still miss the context declaration. I'll see that I will prepare a fix in the next days.

**Comment by ssteinbuss** [comment]
closed by #49
